download.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+# Downloads audio + subtitles in the specified language from YouTube 
+# Converts the audio to 16 KHz mono WAV 
+# 
+# How to use:
+#       Add the YouTube video URLs to download.txt.
+#       Run the script './download.sh <subtitle lang>' (check README for list of languages).
+#       All files will be written to the 'downloads directory.
+#
+# Author: Johni Douglas Marangon - https://github.com/johnidm
+# 
+
+if [ -z $1 ] ; then
+    echo ""
+    echo "Subtitle language is required!"
+    echo "Usage: './download.sh <subtitle lang>'"
+    echo ""
+    exit 1
+fi
+
+download_dir="./downloads"
+
+lang=$1
+
+rm -rf $download_dir
```

Comment:
A confirmation prompt might be added here to let the user know he might lose data downloaded in a previous session. E.g. https://www.shellhacks.com/yes-no-bash-script-prompt-confirmation/

Reply:
Done